src/reducers/dependencies.reducer.test.js

```diff
@@ -0,0 +1,287 @@
+import reducer, { getDependenciesForProjectId } from './dependencies.reducer';
+import {
+  LOAD_DEPENDENCY_INFO_FROM_DISK,
+  ADD_DEPENDENCY_START,
+  ADD_DEPENDENCY_ERROR,
+  ADD_DEPENDENCY_FINISH,
+  UPDATE_DEPENDENCY_START,
+  UPDATE_DEPENDENCY_ERROR,
+  UPDATE_DEPENDENCY_FINISH,
+  DELETE_DEPENDENCY_START,
+  DELETE_DEPENDENCY_ERROR,
+  DELETE_DEPENDENCY_FINISH,
+} from '../actions';
+import { isIterable } from 'core-js';
```

Comment:
Looks like this import is unused?

(I'd also like to avoid using `core-js` if it's not strictly required; or, at least, we should make it a more explicit dependency, since right now it only exists since the babel runtime requires it)

Reply:
Oh, sorry it's mistake with auto-import :)